extensions/typescript/src/typescriptServiceClient.ts

```diff
@@ -328,7 +328,7 @@ export default class TypeScriptServiceClient implements ITypescriptServiceClient
 	}
 
 	private get bundledTypeScriptPath(): string {
-		return path.join(__dirname, '..', 'node_modules', 'typescript', 'lib', 'tsserver.js');
+		return path.join(__dirname, '..', '..', 'node_modules', 'typescript', 'lib', 'tsserver.js');
```

Comment:
Use `require.resolve('typescript/lib/tsserver.js')` instead to avoid the path dependency